components/d2l-filter-dropdown/d2l-filter-dropdown.js

```diff
@@ -19,41 +18,25 @@ class D2LFilterDropdown extends mixinBehaviors([D2L.PolymerBehaviors.FilterDropd
 	static get template() {
 		return html`
 			<style include="d2l-filter-dropdown-styles"></style>
-			<d2l-dropdown class="d2l-filter-dropdown-container">
-				<button
-					class="d2l-dropdown-opener"
-					aria-labelledby="filterText">
-					<span id="filterText">[[_numFiltersText(_numFilters)]]</span>
-					<d2l-icon icon="d2l-tier1:chevron-down"></d2l-icon>
-				</button>
-				<d2l-dropdown-content
-				min-width="[[_minWidth]]"
-				max-width="[[_maxWidth]]"
-					no-padding
-					render-content
-					no-pointer>
-					<div class="d2l-filter-dropdown-content-header">
-						<span>[[localize('filterBy')]]</span>
-						<button hidden$="[[!_numFilters]]" class="d2l-filter-dropdown-clear-button" on-click="_clearFilters">[[localize('clear')]]</button>
-					</div>
-					<div class="d2l-filter-dropdown-tab-selector">
-						<dom-repeat items="[[_filters]]">
+			<d2l-dropdown>
```

Comment:
One too many layers here - we should be able to remove the `d2l-dropdown` (generic opener) since `d2l-dropdown-button-subtle` is an opener.  The generic `d2l-dropdown` opener is only used when we want to define our own opener element and denote it with `class="d2l-dropdown-opener"`.  This is an awkward aspect of the component names, and the fact that so many things are possible.

Reply:
Thanks Dave!  Totally did not realize that, I agree that the naming isn't the most intuitive, but I also didn't read the README close enough

Reply:
Maybe in hindsight we should have named the generic opener `d2l-dropdown-generic`?  Then it might be more obvious that it's not meant to be a wrapper around the other prebuilt openers.